cities/modeling/modeling_utils.py

```diff
@@ -124,50 +124,25 @@ def prep_wide_data_for_inference(
 
     assert f_covariates_joint["GeoFIPS"].equals(intervention["GeoFIPS"])
 
-    # extract data for which intervention and outcome overlap
-    year_min = max(
-        intervention.columns[2:].astype(int).min(),
-        outcome.columns[2:].astype(int).min(),
-    )
-
-    year_max = min(
-        intervention.columns[2:].astype(int).max(),
-        outcome.columns[2:].astype(int).max(),
-    )
-
-    assert all(intervention["GeoFIPS"] == outcome["GeoFIPS"])
-
+    # This is for the downstream variable
     outcome_years_to_keep = [
         year
         for year in outcome.columns[2:]
-        if year_min <= int(year) <= year_max + forward_shift
+        if str(int(year) - forward_shift) in intervention.columns[2:]
     ]
 
```

Comment:
I think this is what intended here but if you want to have the same behavior as before, change it to `outcome_years_to_keep = [year for year in outcome.columns[2:] if year in intervention.columns[2:]]`

Reply:
Thanks!